src/i18nfortests.ts

```diff
@@ -1,34 +1,16 @@
 /* eslint-disable import/no-default-export */
 import i18n from "i18next";
 import { initReactI18next } from "react-i18next";
-import Backend from "i18next-fs-backend";
-import { format as date_fnz_tz_format } from "date-fns-tz";
-import { enCA, frCA } from "date-fns/locale";
+import { i18nconfig } from "./i18n";
+import _ from "lodash";
+
+const i18nconfig_test = _.cloneDeep(i18nconfig); // Copy default configuration
```

Comment:
For one offs like this, I would generally just pull in the module you need instead of all of `lodash` - all you need is this: https://www.npmjs.com/package/lodash.clonedeep

As an alternative, when the object contains just simple values you can deep clone with `JSON.parse(JSON.stringify(i18nconfig))`

Reply:
I keep getting errors when I try to change this - "could not find declaration file for lodash.clonedeep" - won't fix

Reply:
You are introducing thousands of lines of code we aren't using. The lodash maintainers spent a non-trivial amount of time to [modularize lodash](https://www.npmjs.com/search?q=keywords:lodash-modularized) so as to reduce the surface area for bugs to exist on and to keep our bundle size as small as possible.

The error you posted is incomplete - it also includes:

> Try `npm i --save-dev @types/lodash.clonedeep` if it exists or add a new declaration (.d.ts) file containing `declare module 'lodash.clonedeep';`

Run `yarn @types/lodash.clonedeep` and the type will be pulled into the codebase. Note you may need to restart your IDE.

Reply:
Changed to lodash.clonedeep and installed @types/lodash.clonedeep and it works now. Don't think this really matters though. Shopify Polaris lists all of lodash as a dependency.

Reply:
Yes it does. Node modules are installed as a tree hierarchy. Multiple versions of the same dependency can exist in the tree.  Full Lodash weighs in at 75kb, so it's not a tiny dependency. 

https://lexi-lambda.github.io/blog/2016/08/24/understanding-the-npm-dependency-model/

Reply:
It's changed to lodash.clonedeep - it won't let me put this in devDependencies even though I'm using this solely for tests (to clone i18n configuration and change default language to en instead of using the language detector). I think webpack may be smart enough to remove lodash in the build but not sure of this.